spec/railway_ipc/incoming_message_spec.rb

```diff
@@ -24,26 +24,32 @@
 
 RSpec.describe RailwayIpc::IncomingMessage, '#decoded' do
   it 'decodes protobuf binary encoded messages' do
-    incoming_message = described_class.new(stubbed_pb_binary_payload)
+    incoming_message = described_class.new(
+      stubbed_pb_binary_payload,
+      message_format: 'protobuf_binary'
+    )
     expect(incoming_message.decoded).to eq(stubbed_protobuf)
   end
 
```

Comment:
The spec examples below didn't really get deleted, I moved them to the `message_decoders_spec.rb` file.